### ISSUE TITLE
can someone shed some light on how bazel + viewbinding + kotlin work together?

### ISSUE THREAD

**Issue by fisheye-123** [bazelbuild/rules_kotlin#401]

**Comment by restingbull** [comment]
This is a very broad question.
Can you give a better idea of what you need to accomplish? A sample repository would be enough.

**Comment by tdounnyy** [comment]
I think he needs some example?

**Comment by fisheye-123** [comment]
yeah, I am looking for an example which can demonstrate how bazel+kotlin+android viewbinding works. Frankly speaking, after google around, I am actually not sure whether such combination is fully supported by bazel yet.

**Comment by Kernald** [comment]
As far as I understand how ViewBinding works, there needs to be some Bazel-side (or at least rules_android) support for this. I just opened https://github.com/bazelbuild/bazel/issues/13931 to track that.
